include/miral/miral/window_info.h

```diff
@@ -47,7 +47,12 @@ struct WindowInfo
 
     bool is_visible() const;
 
+    /// \remark Since MirAL 5.0
+    /// \deprecated Use Surface::content_size()/Surface::window_size() instead
+    /// @{
+    [[deprecated("use Surface::content_size()/Surface::window_size() instead")]]
```

Comment:
Misleading and unhelpful: This far predates MirAL 5.0; and, `Surface::content_size()`/`Surface::window_size()` are not public APIs

```suggestion
    /// \deprecated Obsolete: Window::size() includes decorations
    /// @{
    [[deprecated("Obsolete: Window::size() includes decorations")]]
```

Reply:
Much better. Thanks.